src/styles/blocks/defaultStyles.scss

```diff
@@ -0,0 +1,18 @@
+body,
+h1,
+h2,
+h3 {
```

Comment:
do you really want to add same font-size and other styles for all of this headers?
you should add those styles only to body because now your h1, h2 and h3 will look identical

Reply:
I tried to style everything in classes but default styles of these tags hinder that, so I just skipped them. How do you think, what will be correct decission for this case?